services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list,
+        username: str,
+        date: str = None
+) -> None:
+    user = User.objects.get(username=username)
```

Comment:
You need to use get_user_model() here with `import from django.contrib.auth import get_user_model`
```suggestion
    user = get_user_model().objects.get(username=username)
```